lib/presentation/journey/todo/todo_list/todo_list_screen.dart

```diff
@@ -11,31 +16,89 @@ class TodoListScreen extends StatefulWidget {
-            IconButton(
-              icon: Icon(Icons.add),
-              onPressed: () =>
-                  Navigator.of(context).pushNamed(RouteList.createTodo),
-            )
-          ],
-        ),
-        body: Center(
-          child: Text('CODE !'),
+  Widget build(BuildContext buildContext) => BlocProvider(
+        create: (BuildContext context) => _todoBloc,
+        child: Scaffold(
+          appBar: AppBar(
+            title: Text('TODOS'),
+            centerTitle: true,
+            actions: [
+              IconButton(
+                icon: Icon(Icons.add),
+                onPressed: () =>
+                    Navigator.of(context).pushNamed(RouteList.createTodo),
+              )
+            ],
+          ),
+          body: Center(
+            child: BlocConsumer<TodoBloc, TodoState>(
+              bloc: _todoBloc,
+              builder: (BuildContext context, state) {
+                switch (state.runtimeType) {
+                  case TodoLoadingState:
+                    return Center(
+                      child: CircularProgressIndicator(),
+                    );
+                    break;
+                  case TodoFetchState:
+                    return ListView.builder(
+                      itemCount: state.todos.data.length,
+                      itemBuilder: (context, index) {
+                        return Card(
+                          elevation: 5,
+                          child: Container(
+                            padding: EdgeInsets.all(10),
+                            height: 50,
+                            child: Row(
+                              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                              children: [
+                                Flexible(
+                                  child: Text(
+                                    '${index + 1}. ${state.todos.data[index].description}',
+                                  ),
+                                ),
+                                Flexible(
+                                  child: Text(
+                                      state.todos.data[index]?.completed ??
+                                              false
+                                          ? 'DONE'
+                                          : 'IN PROGRESS'),
+                                )
+                              ],
+                            ),
+                          ),
+                        );
+                      },
+                    );
+                    break;
+                  default:
+                    return ListView.builder(
+                      itemCount: 1,
+                      itemBuilder: (context, index) {
+                        return Container(
+                          child: Text('null'),
+                        );
+                      },
+                    );
+                }
+              },
+              listener: (BuildContext context, state) {},
```

Comment:
If we arent planning to use listner, then we can skip using blocConsumer. 
BlocBuilder is enough if we just using builder.